client/state/initial-state.js

```diff
@@ -163,15 +141,36 @@ export function persistOnChange( reduxStore, serializeState = serialize ) {
 }
 
 export default function createReduxStoreFromPersistedInitialState( reduxStoreReady ) {
-	if ( config.isEnabled( 'persist-redux' ) && isLoggedIn() && ! isSupportUserSession() ) {
-		localforage
+	const shouldPersist =
+		config.isEnabled( 'persist-redux' ) && isLoggedIn() && ! isSupportUserSession();
+
+	if ( 'development' === process.env.NODE_ENV ) {
+		window.resetState = () => localforage.clear( () => location.reload( true ) );
+
+		if ( shouldAddSympathy() ) {
+			// eslint-disable-next-line no-console
+			console.log(
+				'%cSkipping initial state rehydration to recreate first-load experience.',
+				'font-size: 14px; color: red;'
+			);
+
+			localforage.clear();
+
+			return shouldPersist
+				? reduxStoreReady( persistOnChange( createReduxStore( getInitialServerState() ) ) )
+				: reduxStoreReady( createReduxStore( getInitialServerState() ) );
+		}
```

Comment:
@dmsnell I just tried to use `?flags=force-sympathy` to clear state on staging.  Of course, it didn't work.

What do you think about exporting `window.resetState` in all environments, or all except production?